docs/help/operator.md

```diff
@@ -84,6 +84,34 @@ in your Custom Resource), you need to have two PVs available. By default, each b
 If you specify persistenceEnabled=false in your Custom Resource, the deployed brokers uses ephemeral storage. Ephemeral 
 storage means that that every time you restart the broker Pods, any existing data is lost.
 
+## Configuring logging for the Operator
+
+This section describes how to configure logging for the operator.
+
+The operator image is using zap logger for logging. You can define logging options in the operator yaml before
+deploying the operator as shown in https://github.com/artemiscloud/activemq-artemis-operator/blob/main/deploy/install/110_operator.yaml#L24. This works for manual installing of the operator.
```

Comment:
The operator image is using zap logger for logging. You can set the zap log level editing the container args defined in the operator deployment, i.e.

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    control-plane: controller-manager
  name: activemq-artemis-controller-manager
  spec:
      containers:
      - args:
        - --zap-log-level=error
...
```

You can use edit the operator deployment using the Kubernetes dashboard or the Kubernetes command-line tool, i.e.
```
`sed 's/--zap-log-level=.*/--zap-log-level=error/' deploy/install/110_operator.yaml | kubectl apply -f -`
```

Reply:
+1